legal-api/src/legal_api/reports/business_document.py

```diff
@@ -303,6 +307,8 @@ def _set_parties(self, legal_entity: dict):
                 party["mailingAddress"] = BusinessDocument._format_address(party["mailingAddress"])
             if party.get("deliveryAddress"):
                 party["deliveryAddress"] = BusinessDocument._format_address(party["deliveryAddress"])
+            if not party.get("officer", {}).get("partyType") and not party.get("officer", {}).get("lastName"):
+                party["officer"]["partyType"] = "organization"
         legal_entity["parties"] = party_json
```

Comment:
handles an odd case when the partner of a firm is an organization. Does not handle when there are 2 partner organizations yet.

Example in FM1000051

My initial changes were getting out of hand to handle this fully, so looking for suggestions.